contributor-guide/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,5 +1,82 @@
 = Release Notes
 :navtitle: Release Notes
 
+Writing release notes is crucial for informing users about the changes made in the latest version of a library. While the Boost community does not have a strict format for release notes (some libraries have a _change log_, others incorporate release information into their main documentation), there are however best practices and common structures followed by maintainers. 
```

Comment:
> While the Boost community does not have a strict format for release notes

Kind of. Besides a changelog that

> some libraries have a _change log_, others incorporate release information into their main documentation

there are also the release notes for the boost release. 

Authors have to include their release notes here: https://github.com/boostorg/website/tree/master/feed/history